tox.ini

```diff
@@ -149,4 +149,5 @@ deps =
     codecov
 commands =
     coverage combine
+    coverage xml
```

Comment:
What does this do?

Reply:
precomputes the xml for codecov. probably don't need it, was experimenting.